### ISSUE TITLE
Document format translation

### ISSUE THREAD

**Issue by Martin005** [tolgee/tolgee-platform#1549]
- This feature enables users to import and export documents (DOCX, HTML, ...)

**Comment by jdimeo** [comment]
There is interest in this from our end to consolidate a different provider that is providing AI-based, long form translation

**Comment by JanCizmar** [comment]
Hey! The basic support planned for Q4 :)

https://tolgee.io/roadmap

**Comment by jdimeo** [comment]
Yup, understood! we were discussing it in a meeting so I dropped a note to "vote" for it :-)

**Comment by jdimeo** [comment]
On a related note, we are still using some other tools besides Tolgee for translation here because they support SCORM files natively (e-learning courses). You might want to consider supporting those files along with a feature like this.
